modules/monitoring-configuring-different-alert-receivers-for-default-platform-alerts-and-user-defined-alerts.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-different-alert-receivers-for-default-platform-alerts-and-user-defined-alerts_{context}"]
+= Configuring different alert receivers for default platform alerts and user-defined alerts
+
+You can configure different alert receivers for default platform alerts and user-defined alerts to ensure the following results:
+
+* All default platform alerts are sent to a receiver owned by the team in charge of these alerts.
+* All user-defined alerts are sent to another receiver so that the team can focus only on platform alerts.
+
+You can achieve this by using the `openshift_io_alert_source="platform"` label that is added by the Cluster Monitoring Operator to all platform alerts.
+
+.Prerequisites
+
+ifndef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `dedicated-admin` role.
+endif::openshift-dedicated,openshift-rosa[]
+
+.Procedure
+
+To configure different alert receivers from the CLI:
+
+. Print the Alertmanager configuration into the `alertmanager.yaml` file:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring get secret alertmanager-main --template='{{ index .data "alertmanager.yaml" }}' | base64 --decode > alertmanager.yaml
+----
++
+. Edit the `alertmanager.yaml` file:
++
+[source,yaml]
+----
+...
+route:
+  routes:
+    - matchers:
+        - '"openshift_io_alert_source=platform"'
```

Comment:
it would cause error for alertmanager pod
```
$ oc -n openshift-monitoring logs -c alertmanager alertmanager-main-0
...
ts=2024-04-09T02:50:25.239Z caller=coordinator.go:118 level=error component=configuration msg="Loading configuration file failed" file=/etc/alertmanager/config_out/alertmanager.env.yaml err="bad matcher format: \"openshift_io_alert_source=platform\""
```
```suggestion
        - 'openshift_io_alert_source=platform'
```